container-disc/src/main/sh/vespa-start-container-daemon.sh

```diff
@@ -98,6 +98,8 @@ configure_memory() {
 configure_cpu() {
     if ((jvm_availableProcessors > 0)); then
         cpu_options="-XX:ActiveProcessorCount=${jvm_availableProcessors}"
+    else
+        cpu_options=`nproc`
```

Comment:
This will just produce the number, need to prefix it with `-XX:ActiveProcessorCount=`

Reply:
The office air is too thick .....
Head is heavy...
@jobergum FYI